test/configCases/wasm/path-templates-async/index.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+const path = require('path');
+const fs = require('fs');
+
+it("should use path template for async wasm modules", function() {
+	return import("./module").then(function() {
+        const statsPath = path.join(
+            __dirname,
+            "stats.json"
+        );
+        const stats = JSON.parse(fs.readFileSync(statsPath));
+        let found = false;
+        stats.assets.forEach((asset) => {
+            found = found || !!asset.name.match(
+                /^\w{20}\.\w{20}\.\w{16}\.wasm\.wat\.wat\.wasm$/
+            );
+        });
```

Comment:
```js
        const found = stats.assets.some((asset) => {
            !!asset.name.match(
                /^\w{20}\.\w{20}\.\w{16}\.wasm\.wat\.wat\.wasm$/
            );
        });
```

What about this approach?

Reply:
@jamesgeorge007 Good idea, I did this with the slight change that it should be `.some` rather than `.every` (I'm looking for only one outputted file that matches this reg exp).